index.js

```diff
@@ -154,12 +169,13 @@ const onLivereloadConfig = (slidePipeline, config) => {
 
 on('config', config =>
   minimisted(argv => onConfig(config, argv), {
-    string: ['source', 'out', 'dest', 'port'],
+    string: ['source', 'out', 'dest', 'port', 'openBrowser'],
```

Comment:
You can use `boolean` option here, like:
```js
string: [...],
boolean: ['open-browser'],
```
I think that makes more sense.
ref: https://github.com/substack/minimist#var-argv--parseargsargs-opts

And I prefer '--open-browser' over '--openBrowser'. I think kebab-case is more common in command line options.

Reply:
Awesome. Good call on the kebab case and thanks for showing me how to do the boolean option.

Done!